xc/common/primitives/bufgctrl/bufgctrl.model.xml

```diff
@@ -12,7 +12,7 @@
    <port name="S1" />
   </input_ports>
   <output_ports>
-   <port name="O" is_clock="1" />
```

Comment:
I actually believe we want to leave this as a clock generator, and have explicit SDC specify the output of the BUFG.  This is because the PLL compensation loop complicates things.  The PLL is design to compensate to the output of the BUFG (assuming the BUFG feedback is used).

Reply:
@litghost  Ok. Only thing is that, in this way, we would lose the possibility to have the delay_constant from the input clock to the output clock of the BUFG.